src/app/service/constants/constants.ts

```diff
@@ -24,7 +24,8 @@ export default Object.freeze({
 	DB_COLLECTION_FIRST_QUEST_CONTENT: 'firstQuestContent',
 	DB_COLLECTION_FIRST_QUEST_TRACKER: 'firstQuestTracker',
 	DB_COLLECTION_FIRST_QUEST_POAPS: 'firstQuestPOAPs',
-	MONGODB_URI_PARTIAL: `${process.env.MONGODB_PREFIX}://${process.env.MONGODB_USERNAME}:${process.env.MONGODB_PASS}@${process.env.MONGODB_CLUSTER}/`,
+	MONGODB_URI_PARTIAL: `mongodb+srv://${process.env.MONGODB_USERNAME}:${process.env.MONGODB_PASS}@${process.env.MONGODB_CLUSTER}${process.env.MONGODB_URL_ENDPOINT}`,
```

Comment:
Please stop editing this. Check your `.env` file for the proper syntax.

Reply:
oops